Pod/Classes/iOS/UITapGestureRecognizer+RxGesture.swift

```diff
@@ -26,7 +26,7 @@ import RxCocoa
 public enum UITapGestureRecognizerDefaults {
     public static var numberOfTouchesRequired: Int = 1
     public static var numberOfTapsRequired: Int = 1
-    public static var configuration: ((UITapGestureRecognizer, RxGestureRecognizerDelegate) -> Void)? = nil
+    public static var configuration: ((UITapGestureRecognizer, RxGestureRecognizerDelegate) -> Void)?
```

Comment:
Why did you remove this default value? Does it prevent compilation or it was just enforced by a warning (maybe from swiftlint)?

Reply:
You are right on because of the warning on Swiftlint. Is it an intention to make it explicitly `nil` as Optional types are `nil` implicitly by default. If so I could re-add them back.